src/main/scala/org/scalacheck/commands/Commands.scala

```diff
@@ -215,7 +215,7 @@ trait Commands {
   final def property(threadCount: Int = 1, maxParComb: Int = 1000000): Prop = {
     val suts = collection.mutable.Map.empty[AnyRef,(State,Option[Sut])]
 
-    Prop.forAll(actions(threadCount, maxParComb)) { as =>
+    Prop.forAllNoShrink(actions(threadCount, maxParComb)) { as =>
```

Comment:
Same here, you can use `forAllShrink`.  Otherwise, you're changing the implementation.  Additionally, there is a `Shrink[Actions]` in scope.